.buildkite/commands/configure-git-for-release-management.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
+
+# Git command line client is not configured in Buildkite. Temporarily, we configure it in each step.
+# Later on, we should be able to configure the agent instead.
+curl -L https://api.github.com/meta | jq -r '.ssh_keys | .[]' | sed -e 's/^/github.com /' >> ~/.ssh/known_hosts
+git config --global user.email "mobile+wpmobilebot@automattic.com"
+git config --global user.name "Automattic Release Bot"
+
+# Buildkite is currently using the https url to checkout. We need to override it to be able to use the deploy key.
+git remote set-url origin git@github.com:woocommerce/woocommerce-ios.git
```

Comment:
Same surprise about this as the one I had in WCAndroid (https://github.com/woocommerce/woocommerce-android/pull/10406#discussion_r1427941424), as our checkov policies should already enfore all our `repository` URLs in Buildkite's terraform config files to always use the `git@github.com:` SSH URL format for all pipelines…

And when I look at the terraform config for this repo, it is indeed [already declared to use the SSH url, not the https one](https://github.com/Automattic/buildkite-ci/blob/trunk/src/buildkite/pipelines/woocommerce-ios.tf#L10), and same is true when I look at the Settings for the pipeline in the web UI, so… what gives? 🤔 

<img width="872" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/216089/1e4f6a80-ebca-47ad-92c9-b8aff0f838e9">

Reply:
@AliSoftware Updated in my follow-up PR here: https://github.com/woocommerce/woocommerce-ios/pull/11470/commits/aee6cd0c44b210c1ad3984e45318665cf7d8fa3b